docs/routing/configuration.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Configuration
+name: configuration
+parent: Routing and Navigation
+nav_order: 3
+---
+
+## Routes
+
+Each route defines a name, a pattern, and optionally one or more targets to which to navigate when the route has been matched. In the routes section, you define which patterns are available for navigation.
+
+Targets and routes should match view file to easily track the flow of the navigation.
+
+## Targets
+
+A target defines the view or component that is displayed. It is associated with one or more routes or it can be displayed manually from within the app. Whenever a target is displayed, the corresponding view or component is loaded and added to the aggregation configured with the controlAggregation option of the control.
+
+Route names should be descriptive by itself. Prefer plural form for routes if applicable.
+
+<div class="goodExample" markdown=1>
+
+### Good example
+
+- `home`
+- `productDetails`
+
+</div>
+
+<div class="badExample" markdown=1>
+
+### Bad example
+
+- `page1`
+- `itemDetail`
+
+</div>
+
+## Config
```

Comment:
same comment as in `patterns`: this needs code example to make it "visible" for the reader what the options are all about